package.json

```diff
@@ -39,7 +39,8 @@
     "typescript": "^4.0.5",
     "webpack": "^5.4.0",
     "webpack-bundle-analyzer": "^4.1.0",
-    "webpack-cli": "^4.2.0"
+    "webpack-cli": "^4.2.0",
+    "axios-debug-log": "^0.8.4"
```

Comment:
Don't forget to remove this when done

Reply:
any particular reason? it's a dev dependency that doesn't show up in the final code base and only shows up in dev/unit testing

Reply:
Probably overkill on my end but I'm trying to keep dependencies as trimmed as possible. If it's something more complicated yet useful/essential, then I'm open to evaluate it, but for smaller features that could also be done in a few lines of code I'd rather just make an implementation of it instead of introducing another dependency, even if dev.

Here's an example equivalent you could wrap in a function and call when you want logging for requests/responses:
```
axios.interceptors.request.use(request => {
  console.log('Starting Request', JSON.stringify(request, null, 2));
  return request;
});

axios.interceptors.response.use(response => {
  console.log('Response:', JSON.stringify(response, null, 2));
  return response;
});
```